tools/gui/generators/MDGenerator.h

```diff
@@ -72,12 +73,11 @@ class MDGenerator: public Generator, public InputBase {
 	static const double kelvin_2_mardyn;
 
 protected:
-	Log::Logger* _logger;
-	bool _deleteLogger;
+	std::shared_ptr<Log::Logger> _logger;
 
 	MDGenerator(std::string name);
 
-	virtual ~MDGenerator();
+	virtual ~MDGenerator() {};
```

Comment:
```suggestion
	virtual ~MDGenerator() = default;
```
?

Reply:
Should work...
Do you know of anybody still using those programs in `tools/gui`?

Reply:
honestly no idea.